### PR TITLE
Remove the reference to VPA

### PR DESCRIPTION
**Pull request by vagababov**
We're not really using VPA, so fix the docs, since someone might set it to true and expect something to happen

/assign @dgerd @samodell @markusthoemmes

### CODE REVIEW

File: docs/serving/configuring-autoscaling.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ data:
  container-concurrency-target-default: 100
  container-concurrency-target-percentage: 1.0
  enable-scale-to-zero: true
```
Comment (dgerd) [inline docs/serving/configuring-autoscaling.md]: This seems to be missing a few other options that are in https://github.com/knative/serving/blob/master/config/core/configmaps/autoscaler.yaml

Like scale-to-zero-grace-period, enable-graceful-shutdown, and requests-per-second-target-default. Can you add those as well with this change?

Comment (vagababov) [reply docs/serving/configuring-autoscaling.md]: Hm, I didn't want this to be an overhaul of the docs :)
Also `enable-graceful-shutdown` is not working yet.

Comment (vagababov) [reply docs/serving/configuring-autoscaling.md]: scale-to-zero-grace period is described below.

**Comment by vagababov** [comment]
I did more changes to make it more up to date, but Markus and I need a thorough update for this.

**Comment by dgerd** [comment]
works for me

/lgtm

**Comment by samodell** [comment]
/approved

Thanks for these updates!

**Pull request merged**